README.md

```diff
@@ -1,4 +1,14 @@
-# puppet-module-etcd
+# puppet-etcd
+
+[![Build Status](https://github.com/voxpupuli/puppet-etcd/workflows/CI/badge.svg)](https://github.com/voxpupuli/puppet-etcd/actions?query=workflow%3ACI)
+[![Release](https://github.com/voxpupuli/puppet-etcd/actions/workflows/release.yml/badge.svg)](https://github.com/voxpupuli/puppet-etcd/actions/workflows/release.yml)
+[![Puppet Forge](https://img.shields.io/puppetforge/v/puppet/etcd.svg)](https://forge.puppetlabs.com/puppet/etcd)
+[![Puppet Forge - downloads](https://img.shields.io/puppetforge/dt/puppet/etcd.svg)](https://forge.puppetlabs.com/puppet/etcd)
+[![Puppet Forge - endorsement](https://img.shields.io/puppetforge/e/puppet/etcd.svg)](https://forge.puppetlabs.com/puppet/etcd)
+[![Puppet Forge - scores](https://img.shields.io/puppetforge/f/puppet/etcd.svg)](https://forge.puppetlabs.com/puppet/etcd)
+[![puppetmodule.info docs](http://www.puppetmodule.info/images/badge.png)](http://www.puppetmodule.info/m/puppet-etcd)
+[![AGPL v3 License](https://img.shields.io/github/license/voxpupuli/puppet-etcd.svg)](LICENSE)
```

Comment:
@bastelfreak Umm... AGPL??  The metadata says Apache-2.

Reply:
(and replace https://github.com/voxpupuli/puppet-etcd/blob/628da652d559b048ed55d76b291c5ae11264eea4/LICENSE with the full apache-2 license text for github license detection to work)